using-mobile-extensions/mobile-core/configuration/README.md

```diff
@@ -1,6 +1,169 @@
 # Configuration
 
-The Configuration extension is built into the Mobile Core extension. With the extension APIs, you can generate the necessary events in the Event Hub that provide the configuration options for all installed extensions.
+The Configuration extension is built into the Mobile Core extension. It provides several different APIs for you to setup the configuration either remotely on Launch UI or locally.
+
+## Configure with the Experience Platform Launch App ID
+
+Experience Platform Launch generates a unique environment ID that the SDK uses to retrieve your configuration. This ID is generated when an app configuration is created and published to a given environment. The app is first launched and then the SDK retrieves and uses this Adobe-hosted configuration.
+
+{% hint style="success" %}
+We strongly recommend that you configure the SDK with the Experience Platform Launch environment ID.
+{% endhint %}
+
+After the configuration is retrieved when the app is initially launched, the configuration is stored in local cache. The SDK tries to refresh the configuration every cold launch or when a new session is detected, if there is no change or a network request error occurs while downloading the configuration file, the cached configuration will be used.
+
```

Comment:
Make this two sentences:
The SDK tries to refresh the configuration every cold launch or when a new session is detected, if there is no change or a network request error occurs while downloading the configuration file, the cached configuration will be used.

Like this: 
The SDK tries to refresh the configuration every cold launch or when a new session is detected. If there is no change or a network request error occurs while downloading the configuration file, the cached configuration will be used.